test/integration/integration_test.go

```diff
@@ -95,7 +96,10 @@ func TestConfigImport(t *testing.T) {
 	_, err = os.ReadFile(configPath)
 	require.NotNil(t, err, "reading config file: %v", err)
 
-	runCommand(t, os.Getenv("KUBECTL_AKS"), "config", "import", "-s", subscriptionID, "-g", resourceGroup, "-c", clusterName)
+	runCommand(t, os.Getenv("KUBECTL_AKS"), "config", "import",
+		utils.SubscriptionIDKey, subscriptionID, utils.ResourceGroupKey,
```

Comment:
They are [missing](https://github.com/Azure/kubectl-aks/actions/runs/5553733970/jobs/10142754186#step:10:137) `--` prefix to be interpreted as flags otherwise they are assumed to be `args` and ignored. Meaning that `kubeconfig` is used to import config, making the test green. Some subtle bug :D

Reply:
Great catch!